workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -125,7 +128,8 @@ private static string[] ConstructArgs(string projectPath, string schemaCompilerP
                 $"\"{projectPath}\"",
                 "--",
                 $"--json-dir=\"{ImprobableJsonDir}\"",
-                $"--schema-compiler-path=\"{schemaCompilerPath}\""
+                $"--schema-compiler-path=\"{schemaCompilerPath}\"",
+                $"--worker-json-dir=\"{WorkerJsonRelativePath}\""
```

Comment:
This should be taken relative to `Application.dataPath` see line 77 for what we do with the `SchemaCompilerRelativePath` -> `schemaCompilerPath`